components/bluetooth/test.rs

```diff
@@ -357,10 +359,11 @@ fn create_two_heart_rate_services_device(adapter: &BluetoothAdapter) -> Result<(
 
     let heart_rate_service_empty_2 = try!(create_heart_rate_service(&heart_rate_device_empty, true));
 
-    let _heart_rate_measurement_characteristic =
+    let heart_rate_measurement_characteristic =
         try!(create_characteristic_with_value(&heart_rate_service_empty_1,
                                               HEART_RATE_MEASUREMENT_CHARACTERISTIC_UUID.to_owned(),
                                               vec![0]));
+    try!(heart_rate_measurement_characteristic.set_flags(vec![NOTIFY_FLAG.to_string()]));
```

Comment:
If we don't use the `TwoHeartRateServicesAdapter` in the new tests, why is this set here?